tests/put_template_test.py

```diff
@@ -14,11 +14,13 @@ def __init__(self, mock):
         self.indices = mock
 
 
-class MockPutTemplate(object):
-    def __init__(self, behavior=None):
+class MockTemplateApis(object):
```

Comment:
Not sure what happpened to my earlier comment, so here goes again: Leaving out `object` is the new normal, right?

Reply:
Well, I see it above, but not while in review...

Reply:
Yes, in Python3 you are no longer required to explicitly sub-class `object`.